.github/workflows/prepare-release-pr.yml

```diff
@@ -34,9 +38,9 @@ jobs:
     - name: Prepare release PR (minor/patch release)
       if: github.event.inputs.major == 'no'
       run: |
-        tox -e prepare-release-pr -- ${{ github.event.inputs.branch }} ${{ secrets.chatops }}
+        tox -e prepare-release-pr -- ${{ github.event.inputs.branch }} ${{ secrets.chatops }} --prerelease ${{ github.event.inputs.prerelease }}
```

Comment:
and `--prerelease '${{ github.event.inputs.prerelease }}'`

Reply:
Done, thanks!